src/test/java/payload/Helper.java

```diff
@@ -396,18 +396,28 @@ public static List<Sort> createSortBody(String property, String direction) {
         return sortList;
     }
 
-    public static String getAdminID() {
+    public static String getApplicationID() {
+        ApplicationRegistration applicationRegistration = new ApplicationRegistration();
+        applicationRegistration.setReason("A valid test string must have forty character.");
+        applicationRegistration.setInstitutionId(ConfigurationReader.getProperty("InstitutionID"));
+        while (true) {
+            Response response = InstitutionEndpoints.postRegistrationAdminApplication(applicationRegistration);
+            String applicationID = response.jsonPath().getString("response.id");
+            if (applicationID != null) {
+                return applicationID;
+            }
+        }
+    }
+    public static String getApplicationsID() {
```

Comment:
Birden fazla ID içermediği için `applicationsID` yerine `applicationId` olarak isimlendirmemiz daha iyi olabilir.